SU-SWS/stanford_profile_helper
# D8CORE-4020: fixing the mixing of branches

# READY FOR REVIEW

# Summary
- Adding base styles for SBT

# Review By (Date)
- 3/4

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Pull in the config change.
2. Clear caches.
3. Add teasers on a page.
4. Verify it looks as design noted: 
https://stanfordits.atlassian.net/browse/D8CORE-4020

# Affected Projects or Products
- Teasers for publications

# Associated Issues and/or People
- D8CORE-4020
- https://github.com/SU-SWS/stanford_profile/pull/393
- https://github.com/SU-SOE/soe_basic/pull/71
- https://github.com/SU-SWS/stanford_profile_helper/pull/96


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
